daemon/gluster-blockd.c

```diff
@@ -539,9 +539,17 @@ blockNodeSanityCheck(void)
         return ENOMEM;
       }
     } else {
-      if (GB_ASPRINTF(&global_opts, "%s auto_use_daemon=true", tmp) == -1) {
-        GB_FREE(tmp);
-        return ENOMEM;
+      ret = gbRunner("targetcli set global | grep daemon_use_batch_mode");
```

Comment:
or should s/set/get/ ? though we will get the same result, but it may confusing.

Reply:
@lxbsz 
Maybe get global will have to read the details of each option and display the final output, whereas set global will just display the help. So I thought set will serve the purpose.

Thanks!

Reply:
Both are okay for me.